### PR TITLE
DOC: Better docs for `jax.numpy`: `round`, `around` and `round_`

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Part of #21461

### CODE REVIEW

File: jax/_src/numpy/lax_numpy.py [hunk 1]
```diff
@@ -2644,9 +2644,47 @@ def clip(
...
+  Returns:
+    An array containing the rounded values to the specified ``decimals`` with
+    same shape and dtype as ``a``.
+
+  Note:
+    ``jnp.round`` rounds to the nearest even integer for the values exactly halfway
+    between rounded decimal values.
+
+  See also:
+    - :func:`jax.numpy.floor`: Rounds the input to the nearest integer upwards.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:2670]: downwards

File: jax/_src/numpy/lax_numpy.py [hunk 2]
```diff
@@ -2644,9 +2644,47 @@ def clip(
...
+    - :func:`jax.numpy.ceil`: Rounds the input to the nearest integer downwards.
```
Comment (jakevdp) [inline jax/_src/numpy/lax_numpy.py:2671]: upwards

**Review by jakevdp — CHANGES_REQUESTED** [review]

File: jax/_src/numpy/lax_numpy.py (hunk 1 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

File: jax/_src/numpy/lax_numpy.py (hunk 2 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/lax_numpy.py]: Done

**Review by jakevdp — APPROVED** [review]
Looks great!

**Pull request merged**